src/modules/displayList.js

```diff
@@ -0,0 +1,28 @@
+import enterKey from '../assets/images/return.svg';
+import more from '../assets/images/more.svg';
+
+const enter = new Image();
+enter.src = enterKey;
+const moreDots = new Image();
+moreDots.src = more;
+
+function display(index, toDo) {
+  return `
+    <input class='checkbox' type='checkbox' name='checkbox'>
+    <label class="task" for='checkbox'>${toDo.list[index].description}</label>
+    <img class='dots' src=${moreDots.src}>
+  `;
+}
+
```

Comment:
Try not to use innerHtml, it leads you to XSS, and it makes your code vulnerable